packages/Tag/src/Tag.styles.js

```diff
@@ -0,0 +1,158 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
+import { fontSize } from "@paprika/stylers/lib/helpers";
+import { truncateText } from "@paprika/stylers/lib/includes";
+
+const tagColorStyles = {
+  black: css`
+    background: ${tokens.color.black};
+  `,
+
+  blue: css`
+    background: ${tokens.color.blue};
+  `,
+
+  grey: css`
+    background: ${tokens.color.blackLighten70};
+    color: ${tokens.color.black};
+  `,
+
+  green: css`
+    background: ${tokens.color.greenDarken10};
+  `,
+
+  orange: css`
+    background: ${tokens.color.orangeDarken10};
+  `,
+
+  noRisk: css`
+    background: ${tokens.color.blackLighten70};
+    color: ${tokens.color.black};
+  `,
+
+  lightBlue: css`
+    background: ${tokens.color.blueLighten50};
+    color: ${tokens.color.blueDarken20};
+  `,
+
+  lightOrange: css`
+    background: ${tokens.color.orangeLighten40};
+    color: ${tokens.color.orangeDarken20};
+  `,
+
+  lowRisk: css`
+    background: #299a7a;
+  `,
+
+  mediumRisk: css`
+    background: #c9af28;
+  `,
+
+  highRisk: css`
+    background: #cd3c44;
+  `,
+
+  alert: css`
+    background: none;
+    color: ${tokens.color.orangeDarken10};
+  `,
+};
+
+export const Tags = styled.ul(() => {
+  return css`
+    display: flex;
+    flex-wrap: wrap;
+    line-height: 1;
+    margin: 0;
+    padding: 0;
+  `;
+});
+
+const borderColorStyles = ({ borderColor }) => css`
+  border: 1px solid ${borderColor};
+`;
+
+const tagStyles = ({ tagColor, borderColor }) => css`
+  align-items: center;
+  background: ${tokens.color.blackLighten70};
+  ${borderColor ? borderColorStyles : ""}
+  border-radius: ${tokens.space[0] * 2}px;
+  color: ${tokens.color.white};
+  display: flex;
+  line-height: 1;
+  margin-bottom: 2px;
+  margin-right: ${tokens.spaceSm};
+  max-width: 100%;
+  padding: 2px;
+  ${tagColorStyles[tagColor]}
+
+  [data-pka-anchor="avatar"] {
+    margin-right: ${tokens.space};
+  }
+`;
+
+export const RawButtonTag = styled(RawButton)(tagStyles);
+
+export const Tag = styled.li(tagStyles);
```

Comment:
Can you just do `<div as={condition ? "li" : RawButton}>` in the component?

Reply:
This didn't work for me when working with styled components....

Reply:
> Can you just do <div as={condition ? "li" : RawButton}> in the component?

I think this doesn't work because babel needs to compile this, since `as` is a prop the styled-components plugin uses not react, so doesn't run at runtime. But is just my assumption why is not working.

Reply:
It _can_ work:
https://github.com/acl-services/paprika/blob/master/packages/Button/src/Button.js#L102

Reply:
However, this still allows for an invalid orphaned `<li>` tag with no `<ul>`.  If the consumer doesn't use `<Tags>` or include `onClick` on the `<Tag>`.  

So I think this should actually render a `div`, and the consumer will be obligated to wrap their `<Tag>` children in `<li>` elements if they use `<Tags>`.  

(Maybe that should be named `<TagList>` to be more explicit?)

Reply:
Ok yes there is the common use case that the Tag component is rendered on it's own, in which case I think it should by default be a `div`, so lets go with that. The consumer can then render it `as` something else if needs be (or wrap it in an `li` element alternatively)